musl >= 1.2.2 uses ioctl with TIOCSWINSZ internally. The ioctl
systemcall with this option must be added to the firefox sandbox
systemcall whitelist.

See: https://git.musl-libc.org/cgit/musl/commit/?id=4d5786544bb52c62fc1ae84d91684ef2268afa05

diff -upr firefox-82.0.orig/security/sandbox/linux/SandboxFilter.cpp firefox-82.0/security/sandbox/linux/SandboxFilter.cpp
--- firefox-82.0.orig/security/sandbox/linux/SandboxFilter.cpp	2020-10-31 12:35:45.815282945 +0100
+++ firefox-82.0/security/sandbox/linux/SandboxFilter.cpp	2020-10-31 12:53:16.596449933 +0100
@@ -1619,6 +1619,9 @@ class SocketProcessSandboxPolicy final :
             // ffmpeg, and anything else that calls isatty(), will be told
             // that nothing is a typewriter:
             .ElseIf(request == TCGETS, Error(ENOTTY))
+            // musl uses TIOCSWINSZ internally.
+            // See https://git.musl-libc.org/cgit/musl/commit/?id=4d5786544bb52c62fc1ae84d91684ef2268afa05
+            .ElseIf(request == TIOCSWINSZ, Allow())
             // Allow anything that isn't a tty ioctl, for now; bug 1302711
             // will cover changing this to a default-deny policy.
             .ElseIf(shifted_type != kTtyIoctls, Allow())
